resources/views/patients/edit.blade.php

```diff
@@ -1,160 +1,95 @@
-                            </ul>
+                        <div class="mb-lg-3 mb-md-3 mb-sm-3">
+                            <label for="phone" class="form-label">Phone Number *</label>
+                            <input type="text" class="form-control" id="phone" name="phone"
+                                placeholder="09*********" value="{{ $patient->phone }}">
                         </div>
-                        <div
-                            class="col-lg-4 col-md-6 col-sm-12 text-lg-center text-md-start text-sm-center mt-md-3 mt-sm-3">
-                            <h5>Support Us</h5>
-                            <ul class="list-unstyled mt-3">
-                                <li><a href="#" class="text-decoration-none text-light">Documentation</a></li>
-                                <li><a href="#" class="text-decoration-none text-light">FAQS</a></li>
-                                <li><a href="#" class="text-decoration-none text-light">About Us</a></li>
-                                <li><a href="#" class="text-decoration-none text-light">Contact Us</a></li>
-                            </ul>
+                        <div class="mb-lg-3 mb-md-3 mb-sm-3">
+                            <label for="age" class="form-label">Age *</label>
+                            <input type="number" class="form-control" id="age" name="age" placeholder="age"
+                                value="{{ $patient->age }}">
                         </div>
-
                     </div>
-                    <div class="row mt-3 mb-4">
-                        <div class="col-12 text-sm-center">
-                            <small class="text-light">2023 &copy; CLINIC.ORG &trade;</small>
+                    <div class="col-lg-6 col-md-6">
+                        <div class="mb-lg-3 mb-md-3 mb-sm-3">
+                            <label for="address" class="form-label">Address *</label>
+                            <input type="text" class="form-control" id="address" name="address" placeholder="address"
+                                value="{{ $patient->address }}">
+                        </div>
+                        <div class="mb-lg-3 mb-md-3 mb-sm-3">
+                            <label for="blood_type" class="form-label">Blood Type</label>
+                            <select class="form-select" aria-label="Default select example" id="blood_type"
+                                name="blood_type">
+                                <option>Select Blood Type</option>
+                                <option {{ $patient->blood_type == 'A' ? 'selected' : '' }} value="1">A</option>
+                                <option {{ $patient->blood_type == 'O' ? 'selected' : '' }} value="2">O</option>
+                                <option {{ $patient->blood_type == 'B' ? 'selected' : '' }} value="3">B</option>
+                                <option {{ $patient->blood_type == 'AB' ? 'selected' : '' }} value="4">AB</option>
```

Comment:
I don't know your blood_type ui order?
in your migration file : `$table->enum('blood_type', ['A','B', 'AB','O'])->nullable()->comment('blood_type');`
So, your order is 'A','B', 'AB','O'.